app/database/database.py

```diff
@@ -57,8 +57,8 @@ class Database:
     async def read_in(connection, table, field, elements):
         connection.row_factory = sqlite3.Row
         cursor = connection.cursor()
-        sql = 'SELECT * FROM %s WHERE %s IN (%s)' % (table, field, ','.join(map(str, elements)))
```

Comment:
can you add this and the data_svc/constructor change (in your agent_svc) to a separate PR?